chore(Rs2Walker): isInArea

Sunny-P opened the pull request:
The logic for the isInArea() methods are backwards.
This change doesn't change the actual logic, but changes the method description to properly reflect the logic that's applied.

The overloaded isInArea() method that takes in a centre point and a range previously created a rectangle with larger offsets on the x value, rather than an equally sided square with the calculated corners. This is now changed to create an equally sided square around the centre point. Was there a reason we were creating horizontally favoured rectangle areas?

These pedantic changes better reflect the applied logic without changing the original logic (besides creating an equal-sided area, instead of a horizontally favoured rectangle), so this shouldn't have any affect on the current usages of these methods in other scripts/plugins.

I originally became aware of this flawed logic when I tried to use the `isInArea(WorldPoint... worldPoints)` method to determine if I had the player in a certain area using custom points, instead of the simpler to use overloaded method `isInArea(WorldPoint centerOfArea, int range)`, and of course I ran into problems before looking into how the methods were operating.

Commit:
Changed method description to properly reflect the actual logic applied.

Voxsylvae commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/walker/Rs2Walker.java:
```diff
@@ -1972,9 +1972,9 @@ public static boolean isInArea(WorldPoint... worldPoints) {
      */
     @Deprecated(since = "1.5.5", forRemoval = true)
     public static boolean isInArea(WorldPoint centerOfArea, int range) {
-        WorldPoint nwCorner = new WorldPoint(centerOfArea.getX() + range + range, centerOfArea.getY() - range, centerOfArea.getPlane());
```
should we not change it here to follow the logic of WorldArea, calculating the 
northeast corner
```Java
ne = new WorldPoint(centerOfArea.getX() + radius, centerOfArea.getY() + radius, centerOfArea.getPlane()); 
```
and 
southwest corner 
```Java
sw  =  new WorldPoint(centerOfArea.getX() - radius, centerOfArea.getY() - radius,  centerOfArea.getPlane()); 
```
range -> radius ? 
or better directly use  use  WorldArea

Voxsylvae commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/walker/Rs2Walker.java:
```diff
@@ -1951,15 +1951,15 @@ private static boolean handleWearableTeleports(Transport transport, int itemId)
      */
     public static boolean isInArea(WorldPoint... worldPoints) {
         WorldPoint playerLocation = Rs2Player.getWorldLocation();
-        return playerLocation.getX() <= worldPoints[0].getX() &&   // NW corner x
```
Change the logic here too.
```Java
playerLocation.getX() >= worldPoints[0].getX() &&   // SW corner x
playerLocation.getY() >= worldPoints[0].getY() &&   // SW corner y
playerLocation.getX() <= worldPoints[1].getX() &&   // NE corner x
playerLocation.getY() <= worldPoints[1].getY();     // NE corner Y
```

Sunny-P replied:
> Change the logic here too.
> 
> ```java
> playerLocation.getX() >= worldPoints[0].getX() &&   // SW corner x
> playerLocation.getY() >= worldPoints[0].getY() &&   // SW corner y
> playerLocation.getX() <= worldPoints[1].getX() &&   // NE corner x
> playerLocation.getY() <= worldPoints[1].getY();     // NE corner Y
> ```

Im confused as to what you're referring to here. This was changed to not be so static in its approach.

Commit:
Updated isInArea().
Checks to ensure two world points are used. Cached values from getter methods to keep as performant as possible.
Updated method parameter description again to the more naturally read: NW then SE.
